finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
@cmeubanks - While testing this out, I never saw the third condition met, and thus I didn't see any printouts of "FizzBuzz".  Why would that be, do you think?

Reply:
@mschae16  I realized that the code was running my "Fizz" condition first and being that multiples of 15 are also multiples of 3 the condition tested true for "Fizz" and that is what the output would be for all multiples of 15 (once a condition was "true" the code didn't need to continue testing the other "buzz or "fizzbuzz" conditions for a number).

I have corrected the code so that my first condition in my if statement is testing for multiples of 15, this seems to have corrected the code and now I have all 3 outputs of "Fizz" "Buzz" and "FizzBuzz"

Reply:
@cmeubanks - Awesome, you got it! Thanks for taking the time to review this again.